spec/helper/revenuecat_internal_helper_spec.rb

```diff
@@ -209,6 +196,88 @@
                               "* added a log when `autoSyncPurchases` is disabled (#1749) via aboedo (@aboedo)")
     end
 
+    it 'commits response is cached' do
+      setup_stubs
+      cached_commits_by_old_version = Fastlane::Helper::RevenuecatInternalHelper.instance_variable_get(:@cached_commits_by_old_version)
```

Comment:
Hmm I wouldn't consider this as part of the api of this class so I don't think we should access it. I think we should test this more as part of it's effects, which you're already doing. I mean, when calling the method twice, it should only execute the api calls once. You're already doing that so I think we should just remove these checks IMO.